src/main/kotlin/org/jetbrains/research/testspark/display/TestCasePanel.kt

```diff
@@ -627,6 +648,8 @@ class TestCasePanel(
      */
     fun isRunEnabled() = runTestButton.isEnabled
 
+    fun isGlobalModified(): Boolean = testCase.testCode != initialCodes[0]
+
```

Comment:
Could you add a comment to this method? At first glance, it is not apparent what the method does.

Reply:
What makes the `initialCodes[0]` special compared to the rest of the elements of this array?